src/test/storageperf/tests_gpdb_aoco.sql

```diff
@@ -0,0 +1,6 @@
+-- Test "schedule". List all the tests you want to run here.
+
+CREATE TABLE onecol (i int4) WITH (appendoptimized=true, orientation=column, compresstype=RLE_TYPE, compresslevel=1);
```

Comment:
I don't think this code belongs to this branch/repository. The only code which should live Zedstore branch is which we can merge to upstream Postgres code. This doesn't seem fits that category.

Mostly should have separate repro for such kind to tests/tools, if required.

Reply:
Agreed. Tests against greenplum shouldn't be in this repo. This was just an experiment and I wanted to keep track of what I did. Closing the PR and will move it to a separate performance testing repo in the future.